bundle/manifests/collector_monitoring.coreos.com_v1_prometheusrule.yaml

```diff
@@ -30,6 +30,20 @@ spec:
       labels:
         service: collector
         severity: Warning
+    - alert: CollectorHigh403ForbiddenResponseRate
+      annotations:
+        description: 'Collector: {{ $labels.app_kubernetes_io_instance }}, sink: {{
+          $labels.component_id }} is receiving HTTP 403 Forbidden response rate of
+          {{ $value | printf "%.2f" }} errors/sec over the last 2 minutes, persisting
+          for more than 5 minutes. This could indicate an authorization issue.'
+        summary: 'Collector: {{ $labels.app_kubernetes_io_instance }}, sink: {{ $labels.component_id
+          }} is receiving HTTP 403 Forbidden responses'
+      expr: |
+        sum by (component_id, app_kubernetes_io_instance) (irate(vector_http_client_responses_total{component_kind="sink", status="403"}[2m])) > 0
```

Comment:
Do we need to include namespace here?  The instance = CLF.name right? Does this provide a unique enough check? If changing then the description and summary should also be updated

Reply:
Yes that would be helpful. I will make the change